app/app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -68,6 +69,12 @@ protected void onCreate(Bundle savedInstanceState) {
         final RelativeLayout eegContent = findViewById(R.id.eegContent);
         final RelativeLayout joystickContent = findViewById(R.id.joystickContent);
 
+        JoyStick joyStick = (JoyStick) findViewById(R.id.joy1);
+        joyStick.setPadColor(Color.parseColor("#55ffffff"));
+        joyStick.setPadBackground(R.drawable.icon_round_arrow);
```

Comment:
This doesn't contrast enough against the grey background. If the end-user turns down the brightness of their phone they won't see it.